pkg/scheduler/actions/allocate/allocate.go

```diff
@@ -137,9 +137,13 @@ func (alloc *Action) Execute(ssn *framework.Session) {
 		// because the allocation of job would change the priority of queue among all namespaces,
 		// and the PriorityQueue have no ability to update priority for a special queue.
 		var queue *api.QueueInfo
+		overusedMap := make(map[string]float64)
 		for queueID := range queueInNamespace {
 			currentQueue := ssn.Queues[queueID]
-			if ssn.Overused(currentQueue) {
+			overused, overusedMap := ssn.Overused(currentQueue)
+			_, cpuOverused := overusedMap["MilliCPU"]
+			_, memoryOverused := overusedMap["Memory"]
+			if overused && (cpuOverused || memoryOverused) {
```

Comment:
already used `overused` why need check `cpuOverused` or  `memoryOverused`?

Reply:
> already used `overused` why need check `cpuOverused` or `memoryOverused`?

If basic resources are overused such as CPU and memory, no later jobs in the queue can be allocated resource any more. If scalar resource is overused such as GPU, we should judge what the later jobs in the queue needs. If it only request CPU, it's reasonable to allocate resource for it.